dev-lang/neko/neko-2.3.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Neko is a high-level dynamically typed programming language"
+HOMEPAGE="https://nekovm.org/"
+
+if [[ "${PV}" == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/HaxeFoundation/${PN}.git"
+else
+	# 2.3.0 -> 2-3-0
+	MY_PV="${PV//./-}"
+	SRC_URI="https://github.com/HaxeFoundation/${PN}/archive/refs/tags/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}-${MY_PV}"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="apache gtk2 mysql sqlite ssl"
+
+DEPEND="
+	apache? ( www-servers/apache:2 )
+	gtk2?   ( x11-libs/gtk+:2      )
```

Comment:
gtk2 is not used anymore on gentoo and should be avoided on new packages

Reply:
> gtk2 is not used anymore on gentoo and should be avoided on new packages

I must have missed that announcement. Afaik GTK2 is still used on some major pkgs like pidgin, GIMP or Firefox.

Could you link the announcement?

Reply:
Sorry, I explained myself wrong: there is no official announcement but we are working on dropping the support for gtk2

Reply:
Based on: https://archives.gentoo.org/gentoo-dev/message/158a59d4fb69af254b70a8544b2b6179

I'll remove GTK2 dep from this.

Reply:
https://archives.gentoo.org/gentoo-dev/message/3c8a0bfba99d83566197fa48149b4048
https://bugs.gentoo.org/768993

Reply:
> https://archives.gentoo.org/gentoo-dev/message/3c8a0bfba99d83566197fa48149b4048
> https://bugs.gentoo.org/768993

thx